riscv-rt/link.x.in

```diff
@@ -216,11 +233,11 @@ ASSERT(SIZEOF(.stack) > (_max_hart_id + 1) * _hart_stack_size, "
 ERROR(riscv-rt): .stack section is too small for allocating stacks for all the harts.
 Consider changing `_max_hart_id` or `_hart_stack_size`.");
 
+/* # Other checks */
 ASSERT(SIZEOF(.got) == 0, "
-.got section detected in the input files. Dynamic relocations are not
-supported. If you are linking to C code compiled using the `gcc` crate
-then modify your build script to compile the C code _without_ the
--fPIC flag. See the documentation of the `gcc::Config.fpic` method for
-details.");
+ERROR(riscv-rt): .got section detected in the input files. Dynamic relocations are not
+supported. If you are linking to C code compiled using the `cc` crate then modify your
+build script to compile the C code _without_ the -fPIC flag. See the documentation of
+the `cc::Build.pic` method for details.");
```

Comment:
Would it be possible to support PIC relocation in the future? If it is, this is something I would be interested in working on.

Reply:
It is an interesting research line. Especially for RISC-V 64 targets which currently face issues with static linking (see #153). However, it would need specific code to deal with dynamic linking etc.